js/ad_manager_template.js

```diff
@@ -198,6 +199,15 @@ OO.Ads.manager(function(_, $) {
     this.playerClicked = function(amcAd, showPage) {
     };
 
+    /**
+     * <i>Optional.</i><br/>
+     * Called when the player detects start of ad video playback.
+     * @method AdManager#adVideoPlaying
+     * @public
+     */
+    this.adVideoPlaying = function() {
+
```

Comment:
This is the ad manager template and will be used as a reference for people who want to implement their own ad managers.